app/components/result-list.js

```diff
@@ -29,9 +29,9 @@ export default Ember.Component.extend({
           const xtime = x.get('time');
           const ytime = y.get('time');
 
-          if (xtime === 0) return 1;
-          if (ytime === 0) return -1;
-          if (xtime === ytime) return 0;
+          if (xtime === 0) { return 1;}
```

Comment:
small thing but would you mind fixing this by adding a space after the semi colon to be consistent.

Reply:
Ew!  I can't believe that slipped through.